core/templates/dev/head/components/answer_group_editor.html

```diff
@@ -150,8 +150,11 @@
         </select>
       </span>
     </div>
-    <span ng-if="isCreatingNewState(outcome)">
-      <input type="text" focus-on="newStateNameInputField" ng-model="outcome.newStateName" class="form-control ng-pristine ng-valid ng-scope ng-touched protractor-test-add-state-input" tabindex="0" aria-invalid="false" required>
-    </span>
+    <ng-form name="newStateNameForm">
+      <span ng-if="isCreatingNewState(outcome)">
+        <input type="text" focus-on="newStateNameInputField" name="newStateName" ng-model="outcome.newStateName" class="form-control ng-pristine ng-valid ng-scope ng-touched protractor-test-add-state-input" tabindex="0" aria-invalid="false" ng-pattern="newStateNamePattern" required>
```

Comment:
Sorry -- it looks like this is from a previous commit, but I just realized: can we also get rid of "ng-pristine ng-valid ng-scope ng-touched"? I think these are things that Angular is supposed to set, not that we should write.